cfme/fixtures/v2v.py

```diff
@@ -11,14 +12,32 @@
 from cfme.utils.log import logger
 
 
+FormDataVmObj = namedtuple(
+    'FormDataVmObj', ['form_data', 'vm_obj']
```

Comment:
Consider calling the second attribute `vm_list` since its always a list.  That'll make it more clear/readable having `v2v_fixture_name.vm_list.pop()` instead of `v2v_fixture_name.vm_obj[0]`